src/AmqpBundle/Amqp/DataCollector.php

```diff
@@ -72,33 +72,37 @@ public function getCommands()
      *
      * @return string data collector name
      */
-    public function getName()
+    public function getName(): string
     {
         return $this->name;
     }
 
     /**
-     * Temps total d'exec des commandes.
+     * Return total command execution time.
      *
      * @return float
      */
-    public function getTotalExecutionTime()
+    public function getTotalExecutionTime(): float
     {
-        $ret = 0;
-        foreach ($this->data['commands'] as $command) {
-            $ret += $command['executiontime'];
-        }
-
-        return $ret;
+        return (float) array_sum(array_column($this->data['commands'], 'executiontime'));
     }
 
     /**
-     * Temps moyen d'exec.
+     * Get average execution time.
      *
      * @return float
      */
-    public function getAvgExecutionTime()
+    public function getAvgExecutionTime(): float
+    {
+        return $this->getTotalExecutionTime() ? ($this->getTotalExecutionTime() / \count($this->data['commands'])) : (float) 0;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function reset()
     {
-        return ($this->getTotalExecutionTime()) ? ($this->getTotalExecutionTime() / count($this->data['commands'])) : 0;
+        $this->data = [];
```

Comment:
could be `$this->data = ['commands' => []];` and remove the next line

Reply:
True, let me fix that quickly.

Reply:
Changed the line. Thanks for catching that :)